docs/conf.py

```diff
@@ -166,9 +166,8 @@ def reset_napari_theme(gallery_conf, fname):
     settings.appearance.theme = 'dark'
     qtgallery.reset_qapp(gallery_conf, fname)
 
-
 sphinx_gallery_conf = {
-    'examples_dirs': '../examples',  # path to your example scripts
+    'examples_dirs': '../../napari/examples',  # path to your example scripts
```

Comment:
I think you need to make sure to merge main here, else building docs on napari/napari PR will break again.

Reply:
I strongly prefer to make it more configurable. People could have different structures of their repositories, and it should be possible to build docs. 
Maybe some environment variable with the path to the napari repository (with default to the currently proposed path).

Reply:
An environment variable sounds like a good option but I am not sure how it would work? (maybe just something I don't know about).

Not sure if this is helpful but Sphinx allows overriding conf.py values from the command line (see last note [here](https://sphinx-gallery.github.io/stable/configuration.html#parsing-and-executing-examples-via-matching-patterns)) - thus the value in the `conf.py` file is effectively the default and the command in the makefile could be made configurable somehow....? E.g., to set `examples_dirs` you can do:
 
`sphinx-build -D sphinx_gallery_conf.examples_dirs=<my_local_path>`

Reply:
Definitely better - thanks @lucyleeow , I'll include this suggestion in this PR.